<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dict_edit</class>
 <widget class="QMainWindow" name="dict_edit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>560</width>
    <height>422</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="dict_edit_label_dict">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>72</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>单词</string>
    </property>
   </widget>
   <widget class="QLabel" name="dict_edit_label_group">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>72</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>分组</string>
    </property>
   </widget>
   <widget class="QLabel" name="dict_edit_label_mask">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>72</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>音标</string>
    </property>
   </widget>
   <widget class="QLabel" name="dict_edit_label_explain">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>180</y>
      <width>72</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>释义</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="dict_edit_edit_dict">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>50</y>
      <width>281</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="dict_edit_button_dict">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>50</y>
      <width>121</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color:transparent;
	border:1px solid gray;
	border-radius:2px;

}</string>
    </property>
    <property name="text">
     <string>从单词本中删除</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="dict_edit_edit_mask">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>130</y>
      <width>451</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
   </widget>
   <widget class="QComboBox" name="dict_edit_combox_group">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>90</y>
      <width>451</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QComboBox{
	background-color:white;
	border:1px solid gray;

}

</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="dict_edit_edit_explain">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>180</y>
      <width>451</width>
      <height>131</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="dict_edit_checkbox">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>330</y>
      <width>171</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>加入复习计划</string>
    </property>
   </widget>
   <widget class="QPushButton" name="dict_edit_button_sure">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>360</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#dict_edit_button_sure{
        background-color: rgb(242, 58, 58);
        color:#fff;
		
        border-radius:5px;
}

QPushButton#dict_edit_button_sure::hover{
        background-color: rgb(247, 84, 84);
        color:#fff;
        border-radius:5px;
        }





</string>
    </property>
    <property name="text">
     <string>确定</string>
    </property>
   </widget>
   <widget class="QPushButton" name="dict_edit_button_cancel">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>360</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	border:1px solid gray;
	background-color:rgb(242,242,242);
	border-radius:2px;

}</string>
    </property>
    <property name="text">
     <string>取消</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>72</width>
      <height>15</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>编辑单词</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_close">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>10</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
    <property name="toolTip">
     <string>关闭</string>
    </property>
    <property name="styleSheet">
     <string notr="true">#button_close{
	
	border-image: url(:/images/close.png);
	border:none;
}

#button_close::hover{
	
	border-image: url(:/images/close-hover.png);
	border:none;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
